src/windows_toasts/__init__.py

```diff
@@ -1,3 +1,11 @@
+import platform
+
+# We'll assume it's Windows since if it's on another OS it should be self-explanatory
+MIN_VERSION = 10240
+if platform.system() == "Windows" and (osVersion := int(platform.version().split(".")[2])) < MIN_VERSION:
```

Comment:
There's many way to get the version, it seems all of them have their special things.
This one seems good to me 👍

https://docs.python.org/3/library/sys.html#sys.getwindowsversion
https://docs.python.org/3/library/platform.html#platform.release

Reply:
Although that version number is the right one? They started with 10.000?

```
platform.version().split(".")
['10', '0', '22621']
```

Reply:
The build number is the clearest number that can be used, only ascending chronologically. Windows 7 for example is marked NT 6.1, the latest Windows release is 23H2, etc.